app/schema.py

```diff
@@ -18,9 +20,11 @@ class Mentor(BaseModel):
     industry_knowledge: bool
     pair_programming: bool
     commitment: bool
-    referred_by: constr(max_length=255)
+    referred_by: Optional[constr(max_length=255)]
     other_info: Optional[constr(max_length=2500)]
-    validate_status: constr(max_length=255)
+    validate_status: Literal['approved', 'rejected', 'pending']
+    is_active: bool
+    accepting_new_mentees: bool
 
```

Comment:
Have we considered setting defaults for `is_active` and `accepting_new_mentees`?